lib/src/model/directus_file.dart

```diff
@@ -1,13 +1,21 @@
-class DirectusFile {
+import 'package:directus_api_manager/src/model/directus_data.dart';
+
+class DirectusFile extends DirectusData {
   static String? baseUrl;
-  final String id;
-  final String? title;
 
-  DirectusFile.fromJSON(Map<String, dynamic> jsonData)
-      : id = jsonData["id"],
-        title = jsonData["title"];
+  String? get title => getValue(forKey: "title");
+  String? get type => getValue(forKey: "type");
+  DateTime get uploadedOn => getDateTime(forKey: "uploaded_on");
+  int? get fileSize => getValue(forKey: "filesize");
+  int? get width => getValue(forKey: "width");
+  int? get height => getValue(forKey: "height");
+  int? get duration => getValue(forKey: "duration");
+  String? get description => getValue(forKey: "description");
+
+  DirectusFile(Map<String, dynamic> rawReceivedData) : super(rawReceivedData);
 
-  const DirectusFile(this.id, {this.title});
```

Comment:
This constructor is very useful for quickly creating DirectusFile objects when we only have the `id` available.
We only need the id to build the file download url so we should keep some way to use DirectusFile with this minimum data